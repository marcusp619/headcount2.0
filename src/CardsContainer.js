import React from 'react';
import Card from './Card'

const CardsContainer = ({schoolData}) => {

  const listLocations = Object.keys(schoolData);
  const locationCards = listLocations.map((location, i) =>
    <Card 
      {...schoolData[location] } key={Date.now() + i}
    />)
  return (
    <div className = "cards-container">
      { locationCards }
    </div>
  )
}


export default CardsContainer;