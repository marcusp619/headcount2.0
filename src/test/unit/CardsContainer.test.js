import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import CardsContainer from '../../../src/CardsContainer';


describe('CardsContainer', () => {
  it('should render one-eight-one cards', () => {
    const wrapper = shallow(<CardsContainer schoolData={[]} />);    
    
    expect(wrapper).toMatchSnapshot();
  })

})
