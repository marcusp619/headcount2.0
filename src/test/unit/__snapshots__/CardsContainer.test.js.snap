// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardsContainer should render one-eight-one cards 1`] = `
<div
  className="cards-container"
/>
`;
